db/seeds.rb

```diff
@@ -29,3 +29,24 @@
     )
   end
 end
+
+def create_mail_method
+  Spree::MailMethod.destroy_all
+
+  CreateMailMethod.new(
+    environment: Rails.env,
+    preferred_enable_mail_delivery: true,
+    preferred_mail_host: ENV.fetch('MAIL_HOST'),
+    preferred_mail_domain: ENV.fetch('MAIL_DOMAIN'),
+    preferred_mail_port: ENV.fetch('MAIL_PORT'),
+    preferred_mail_auth_type: 'login',
+    preferred_smtp_username: ENV.fetch('SMTP_USERNAME'),
+    preferred_smtp_password: ENV.fetch('SMTP_PASSWORD'),
+    preferred_secure_connection_type: 'None',
```

Comment:
Shouldn't we allow a secure option here? I am using TLS actually with my Gmail account. 
Which is the mail system advised by OFN? Couldn't find information from the wiki, I could add it if necessary.

Reply:
All these settings come from https://github.com/openfoodfoundation/ofn-install/wiki/Sending-emails-from-OFN. Specifically, that depends on the mail provider and it looks like mandrill and others require this.

Reply:
If we ever need something different, easy. PR adding another env var.

Reply:
Ok, great, I was looking to the wrong wiki. Then it is all good for me.